rbac/providers/common/admin_setup.py

```diff
@@ -71,9 +125,24 @@ def add_admin_accounts():
             ("http://rbac-server:8000/api/roles/{}/members".format(role_next_id)),
             json=add_user,
         )
+        add_role_member_response_json = add_role_member_response.json()
         if add_role_member_response.status_code >= 300:
```

Comment:
If we are checking for the exact 200 response for user creation, we should do the same here when the Admin is added to the NextAdmins role.